README.md

```diff
@@ -588,6 +588,12 @@ var signIn = new OktaSignIn(config);
     helpSupportNumber: '(123) 456-7890'
     ```
 
+- **signOutUrl:** url to redirect when a user clicks the sign out link. If not provided, the widget will navigate to primary auth.
```

Comment:
Am okay with that and we can shift later when we change all of them at the same time. General idea is that we want everything to be as flexible as possible.